app/screens/shared/date-header/DateHeaderComponent.js

```diff
@@ -32,18 +32,22 @@ class DateHeaderComponent extends Component {
         id="date-header"
         style={{ textAlign: 'center' }}
       >
-        <button
-          onClick={() => onChange(addToDate(date, -1))}
-          style={Object.assign({}, iconButtonStyles, { float: 'left' })}
-        >
-          <Glyphicon glyph="chevron-left" />
-        </button>
-        <button
-          onClick={() => onChange(addToDate(date, 1))}
-          style={Object.assign({}, iconButtonStyles, { float: 'right' })}
-        >
-          <Glyphicon glyph="chevron-right" />
-        </button>
+        {onChange && (
+          <button
+            onClick={() => onChange(addToDate(date, -1))}
```

Comment:
I know this is not related to the changes on this PR, but would say it's not so intuitive to use an onChange this way. I would rather have two onChange functions in the prop, and pass the prop like onClick={buttonNameOnClick}

Maybe it would be clear just changing onChange name for something a more descriptive.